picard/formats/id3.py

```diff
@@ -177,6 +180,18 @@ class ID3File(File):
     __rtranslate_freetext = dict([(v, k) for k, v in __translate_freetext.items()])
     __translate_freetext['writer'] = 'writer'  # For backward compatibility of case
 
+    # Freetext fields that are loaded case-insensitive
+    __rtranslate_freetext_ci = {
+        'replaygain_album_gain': 'REPLAYGAIN_ALBUM_GAIN',
+        'replaygain_album_peak': 'REPLAYGAIN_ALBUM_PEAK',
+        'replaygain_album_range': 'REPLAYGAIN_ALBUM_RANGE',
+        'replaygain_track_gain': 'REPLAYGAIN_TRACK_GAIN',
+        'replaygain_track_peak': 'REPLAYGAIN_TRACK_PEAK',
+        'replaygain_track_range': 'REPLAYGAIN_TRACK_RANGE',
+        'replaygain_reference_loudness': 'REPLAYGAIN_REFERENCE_LOUDNESS',
+    }
+    __translate_freetext_ci = dict([(b.lower(), a) for a, b in __rtranslate_freetext_ci.items()])
```

Comment:
May be introduce a small utility function to do that, as it is repeated at https://github.com/metabrainz/picard/pull/1263/files#diff-4909d6dcdd5d880b988729ebd5d5586dR138

Reply:
I'd rather rework all the formats and unify the tag mapping handle in a separate PR then to blow up this PR. In this process we could extract utility functions for dict reversal.